enterprise/frontend/test/metabase-enterprise/custom_drill_through/drill_through.cy.spec.js

```diff
@@ -6,7 +6,8 @@ import {
   describeWithToken,
 } from "__support__/cypress";
 
-describeWithToken("drill through", () => {
+// [quarantine] flaky
+describe.skip("drill through", () => {
```

Comment:
It should fail with the regular `describe` and I think a simple git log/blame can give a hint as to how to revert it back.
Or is there a better alternative?

Reply:
Either a comment could help, or if you know exactly which test is failing you can skip just that one.

Reply:
Added a TODO note and commented out `desribeWithToken` from the imports. I am actually surprised `yarn lint-eslint` didn't fail there because it always fails when there is an import that is not in use.

Reply:
I added `describeWithToken` to auto-skip EE-specific tests if there's no EE token present. The thinking was that there'd be a unified test run across EE/CE functionality. CI would have the token loaded and EE tests would run, but if an external contributor didn't have a token they'd still be able to run tests.

That thinking might be outdated. If we split up EE/CE tests by file path and have separate runs, we don't need that helper.

If we do want to keep these, I agree a comment would help reverting for now. We could make `describeWithToken.skip` work pretty easily I think.

Reply:
Good idea on adding clarifying comment, will do that @nemanjaglumac @paulrosenzweig

Reply:
I already did in https://github.com/metabase/metabase/pull/13683/commits/1060974104e49beba88c1cd461ce5233c5423c6c. Hope you don't mind?

Reply:
Not at all. Thanks @nemanjaglumac 👍